test/axes/baseAxisTests.ts

```diff
@@ -29,11 +29,11 @@ describe("BaseAxis", () => {
     var verticalAxis = new Plottable.Axis(scale, "right");
     verticalAxis.renderTo(svg);
 
-    var expectedWidth = verticalAxis.tickLength() + verticalAxis.margin(); // tick length and margin by default
+    var expectedWidth = verticalAxis.innerTickLength() + verticalAxis.margin(); // innerTickLength and margin by default
```

Comment:
The only extra information is that this is the default width, hence let's rename the variable to `defaultWidth` and remove the comment.